packages/dx-react-grid/src/plugins/local-sorting.jsx

```diff
@@ -1,21 +1,20 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Getter, PluginContainer } from '@devexpress/dx-react-core';
-import { sortedRows } from '@devexpress/dx-grid-core';
+import { sortedRows, getColumnExtension } from '@devexpress/dx-grid-core';
 
 const pluginDependencies = [
   { pluginName: 'SortingState' },
 ];
 
 export class LocalSorting extends React.PureComponent {
   render() {
-    const { getColumnCompare } = this.props;
+    const { columnExtensions } = this.props;
+    const getColumnCompare = columnName =>
```

Comment:
Should we change `getColumnCompare` to `columnCompare`?

Reply:
I left the `getColumnCompare` name in order to make it consistent across local processing plugins.